include/cvc5/cvc5_proof_rule.h

```diff
@@ -1433,6 +1429,8 @@ enum ENUM(ProofRule)
    *   \inferrule{(t_1\cdot t_2) = (s_1 \cdot s_2),\, \mathit{len}(t_2) =
    *   \mathit{len}(s_2)\mid \top}{t_2 = s_2}
    *
+   * where :math:`t_2` and :math:`s_2` is a concatenation of zero or more strings.
```

Comment:
If you agree with me above then this could change to a chain of concatenations as well

Reply:
Updated